src/index.html

```diff
@@ -22,6 +22,375 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="#"
+        />
+      </a>
+      <nav class="nav">
```

Comment:
Consider adding adding space between elements in one level of depths
```suggestion
      </a>
      
      <nav class="nav">
```

Fix this in all HTML structure